bridge/server/app.ts

```diff
@@ -134,7 +137,7 @@ async function setBasicAUTH(app: Express, auth: AuthConfig): Promise<void> {
     userIP = userIP instanceof Array ? userIP[0] : userIP;
 
     if (userIP && isIPThrottled(userIP, auth)) {
-      log.error('Request limit reached');
+      log.error(`Request limit reached for IP ${userIP}`);
```

Comment:
Is IP logging allowed? GDPR wise?

Reply:
I know there is an exception for data collection if it is for legitimate interest, which I guess this case fits since someone is trying to misuse the product. However, better be safe than sorry :) I will remove it to be on the safe side